imgp

```diff
@@ -60,6 +62,23 @@ count = 0  # successful conversion count
 count_lowres = 0  # no-op count due to low resolution
 
 
+def try_int(value: str) -> int:
+    """
+    :param value: str
+    :return: integer of value or 0 on exception
+    >>> try_int('123')
+    123
+    >>> try_int('ab30')
+    0
+    >>> try_int('abc')
+    0
+    """
+    try:
+        return int(value)
+    except ValueError:
+        return 0
```

Comment:
We should gracefully exit the program in this case. it's not our job to fix user input.